zone/guild.cpp

```diff
@@ -409,3 +512,349 @@ void Client::GuildChangeRank(const char* name, uint32 guild_id, uint32 oldrank,
 	SendGuildMembers(guild_id, true);
 }*/
 
+void Client::SendGuildMembersList()
+{
+	auto   guild_name = guild_mgr.GetGuildName(GuildID());
+	uint32 len;
+	uint8 *data = guild_mgr.MakeGuildMembers(GuildID(), guild_name, len);
+	if (data == nullptr) {
+		return;
+	}
+
+	auto outapp = new EQApplicationPacket(OP_GuildMemberList);
+	outapp->size    = len;
+	outapp->pBuffer = data;
+	data = nullptr;
+
+	LogGuilds("Sending OP_GuildMemberList of length [{}]", outapp->size);
+
+	FastQueuePacket(&outapp);
+
+	SendGuildMOTD();
+	SendGuildChannel();
+	SendGuildURL();
+
+	SetGuildListDirty(false);
+}
+
+void Client::SendGuildMemberAdd(
+	uint32 guild_id,
+	uint32 level,
+	uint32 class_,
+	uint32 rank_,
+	uint32 spirit,
```

Comment:
Rename this one too to match a struct field rename

Reply:
Updated